src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -105,13 +135,13 @@ public void start(Stage primaryStage) throws Exception {
         // menuBar.getMenus().addAll(menuConvo);
```

Comment:
nit: Unless you're planning on using this code later, probably better to delete it.